src/test/java/core/basesyntax/service/fruittransactionservice/FruitTransactionProcessorImplTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.service.fruittransactionservice;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.FruitsStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operationsservice.BalanceFruitOperationHandler;
+import core.basesyntax.service.operationsservice.FruitOperationHandler;
+import core.basesyntax.service.operationsservice.PurchaseFruitOperationHandler;
+import core.basesyntax.service.operationsservice.ReturnFruitOperationHandler;
+import core.basesyntax.service.operationsservice.SupplyFruitOperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FruitTransactionProcessorImplTest {
+    private Map<FruitTransaction.Operation, FruitOperationHandler> operationsMap = new HashMap<>();
+    private FruitDao fruitDao = new FruitDaoImpl();
+    private OperationStrategy operationStrategy = new OperationStrategyImpl(operationsMap);
+    private FruitTransactionProcessor fruitTransaction
+            = new FruitTransactionProcessorImpl(operationStrategy);
+
+    @Test
+    public void makeDailyFruitsUpdateIncludingAllTypesOfOperations() {
+        operationsMap.put(FruitTransaction.Operation.BALANCE,
+                new BalanceFruitOperationHandler(fruitDao));
+        operationsMap.put(FruitTransaction.Operation.PURCHASE,
+                new PurchaseFruitOperationHandler(fruitDao));
+        operationsMap.put(FruitTransaction.Operation.RETURN,
+                new ReturnFruitOperationHandler(fruitDao));
+        operationsMap.put(FruitTransaction.Operation.SUPPLY,
+                new SupplyFruitOperationHandler(fruitDao));
+
+        FruitTransaction bananaTransaction = new FruitTransaction();
+        bananaTransaction.setFruit("apple");
+        bananaTransaction.setQuantity(120);
+        bananaTransaction.setOperation(FruitTransaction.Operation.BALANCE);
+        FruitTransaction appleTransaction = new FruitTransaction();
+        appleTransaction.setFruit("apple");
+        appleTransaction.setQuantity(20);
+        appleTransaction.setOperation(FruitTransaction.Operation.PURCHASE);
+        FruitTransaction orangeTransaction = new FruitTransaction();
+        orangeTransaction.setFruit("apple");
+        orangeTransaction.setQuantity(40);
+        orangeTransaction.setOperation(FruitTransaction.Operation.RETURN);
+        FruitTransaction kiwiTransaction = new FruitTransaction();
+        kiwiTransaction.setFruit("apple");
+        kiwiTransaction.setQuantity(10);
+        kiwiTransaction.setOperation(FruitTransaction.Operation.SUPPLY);
+        List<FruitTransaction> listOfTransactions
+                = List.of(bananaTransaction, appleTransaction, orangeTransaction, kiwiTransaction);
```

Comment:
```suggestion
        List<FruitTransaction> listOfTransactions = getTransactions();
```
method is too long, create a separate method to create transactions there